src/typed-method-types/workflows/triggers/mod.ts

```diff
@@ -0,0 +1,44 @@
+import { BaseMethodArgs, BaseResponse } from "../../../types.ts";
+import { EventTrigger } from "./event.ts";
+import { ScheduledTrigger } from "./schedule.ts";
+import { ShortcutTrigger } from "./shortcut.ts";
+import { WebhookTrigger } from "./webhook.ts";
+
+// TODO: Export to SDK
+export const VALID_TRIGGERS = {
+  event: "event",
+  scheduled: "scheduled",
+  shortcut: "shortcut",
+  webhook: "webhook",
+} as const;
```

Comment:
This is an area where I'd love other peoples opinions on how we should surface these valid triggers in the SDK. As written, the only way these will be surfaced to the SDK is via typeahead. This is going to come up again for a few different parts of triggers.

I think that most users wouldn't expect to import these from the API Client and instead expect to import from the SDK, but importing these to the SDK just to export them again introduces a dependency on the API client.

Options:
- Introduce the dependency and import the API library in the SDK
- Solve discoverability through documentation and keep exports in the API library
- Do nothing and lean on typeahead

Reply:
> I assume we'll need to have the SDK depend on the API library to implement that hook.

FWIW since the developer is working out of their own project which already includes both libraries both options are viable, it's just be a matter of what library they import into their trigger definition file:
```ts
import { TriggerTypes } from "deno-slack-sdk/mod.ts";
// vs
import { TriggerTypes } from "deno-slack-api/mod.ts";
```